locales/en-US/app.ftl

```diff
@@ -14,16 +14,31 @@
 -firefox-brand-name = Firefox
 -profiler-brand-name = Firefox Profiler
 
-##
+## AppHeader
```

Comment:
@canova I have added these comments above starting of each component section. Let me if you feel like it's not necessary.

Reply:
They are not really descriptive since they only include the component name which they all include already. But I think it's an improvement compared to the last one. Let's keep them.

Reply:
Yeah, probably we should add short descriptions indeed. But that can be in a future PR.